test/testdata/infer/autocorrect_no_block.rb

```diff
@@ -6,22 +6,22 @@ def takes_no_block(x: 0)
 end
 
 takes_no_block(x: 0) do end
-#                   ^^^^^^^ error: does not take a block
+#                    ^^^^^^ error: does not take a block
```

Comment:
the error highlight no longer highlights the ` ` character, but the autocorrect still removes it. this is in line with how our "too many args" errors+fixes work.